docs/en/data_types/special_data_types/interval.md

```diff
@@ -0,0 +1,74 @@
+# Interval {#data-type-interval}
+
+Represents time and date intervals. The resulting type of the `INTERVAL` operator.
+
+!!!warning "Warning"
+    You can't use the `Interval` data type for storing values in tables.
+
+Structure:
+
+- Unsigned integer value of interval.
```

Comment:
* value -> length?
* from this explanation it's not obvious that each interval value means "this integer multiplied by the unit from interval time"